### ISSUE TITLE
Dockerfile: specify -nv --show-progress --progress=bar:force:noscroll option for wget downloads

### ISSUE THREAD

**Issue by ptitzler** [IBM/MAX-Image-Segmenter#21]
When downloading large assets from COS it can take several minutes for the download to complete. The currently used wget option `wget -nv` does not display the download progress. We should use 
`wget -nv --show-progress --progress=bar:force:noscroll` instead. The output looks as follows:

````
Step 3/11 : RUN wget -nv --show-progress --progress=bar:force:noscroll http://... ...
---> Running in e9a4783f37f1
data.tar.gz          83%[===============>    ]   3.01G  6.39MB/s    eta 44s 
```

**Comment by bdwyer2** [comment]
PR #22

**Comment by bdwyer2** [comment]
Closed by #22
